src/utils/beneficial.owner.other.ts

```diff
@@ -64,7 +65,9 @@ export const postBeneficialOwnerOther = async (req: Request, res: Response, next
     const session = req.session as Session;
     setApplicationData(session, data, BeneficialOwnerOtherKey);
 
-    await saveAndContinue(req, session);
+    if (registrationFlag || FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME){
```

Comment:
You can pass the registrationFlag into saveAndContinue and then do the logic in one place instead of everywhere.
function saveAndContinue(req, session, isRegistration) {
    if ((isRegistration && sActiveFeature(FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) || (!isRegistration && isActiveFeature(FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME)) {
       await updateOverseasEntity(req, session);
   }
}